benchmark/assert/deepequal-prims-and-objs-big-loop.js

```diff
@@ -22,6 +22,7 @@ const primValues = {
   'empty_object': {},
   'regexp': /abc/i,
   'date': new Date(),
+  'invalidDate': new Date('foo'),
```

Comment:
I am not sure if we want to add the entries to our benchmarks yet, since this would just always fail when comparing it with older versions.

Reply:
The other option is to wait a release to include, but I don't thing is worthy, so I would keep it